my_list.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'my_enumerable'
+
+class MyList
+  include MyEnumerable
+
+  def initialize(*list)
+    @list = list
+  end
+
+  def each
+    i = 0
+    while i < @list.length
+      yield @list[i]
```

Comment:
- [ ] You should have a print after the def to make sure that the code is returning the correct values.

![Capture web_18-12-2021_94139_gitlab com](https://user-images.githubusercontent.com/61562626/146635184-afc37b2c-90a3-4070-b448-5de7fecb81cf.jpeg)